data_pipeline/generic_web_api/generic_web_api_config.py

```diff
@@ -84,9 +85,27 @@ def __init__(
         page_number_param = configurable_parameters.get(
             "pageParameterName", None
         )
+        offset_param = configurable_parameters.get(
+            "offsetParameterName", None
+        )
         page_size_param = configurable_parameters.get(
             "pageSizeParameterName", None
         )
+        result_sort_param = configurable_parameters.get(
+            "resultSortParameterName", None
+        )
+        result_sort_param_value = configurable_parameters.get(
+            "resultSortParameterValue", None
+        )
+        compose_able_static_parameters = (
```

Comment:
The name looks slightly odd because `composeable` is one word.

To me it seems a bit hard to follow maybe because `compose` is a bit generic. I can work it out though. But I think if I had to look at it again in two months I'd probably scratch my head again. I guess those parameters are the ones that end up as URL query parameters.